### ISSUE TITLE
boinc-client does not respect/use LSB startup values.

### ISSUE THREAD

**Issue by romw** [BOINC/boinc#1134]
**Reported by shirish on 9 Jan 41935103 02:58 UTC**
Hi all, 

I have installed boinc-client but while installing
boinc gave couple of warnings. I have aptitude give a bit more verbose
output when installing hence saw those. Do not have any idea whether
its an important warning or not.

```
D000001: deferred_configure updating conffilesupdate-rc.d: warning:
boinc-client start runlevel arguments (2 3 4 5) do not match LSB
Default-Start values (4 5)update-rc.d: warning: boinc-client stop
runlevel arguments (0 1 6) do not match LSB Default-Stop values (0 1 2
3 6)
```

Can you please look it up and see. Please let me know if any more help
is needed ?

One another thing, I got this privacy message while trying to file the
bug in reportbug about whether I choose to include or not the info.
that boinc-client/remove_boinc_dir: false is set this way or not. I have no
idea what boinc-client/remove_boinc_dir: false does and if that is the
default setup or not.

Also see the stuff at - Configuration Files:/etc/boinc-client/gui_rpc_auth.cfg [13](Errno)
Permission denied: u'/etc/boinc-client/gui_rpc_auth.cfg'

Looking forward to comments and suggestions.

-- Package-specific info:-- Contents of /etc/default/boinc-client:#
This file is /etc/default/boinc-client, it is a configuration file for
the# /etc/init.d/boinc-client init script.
# Set this to 1 to enable and to 0 to disable the init script.

ENABLED="1"
# Set this to 1 to enable advanced scheduling of the BOINC core client

and# all its sub-processes (reduces the impact of BOINC on the
system's performance).
SCHEDULE="1"
# The BOINC core client will be started with the permissions of this

user.
BOINC_USER="boinc"
# This is the data directory of the BOINC core

client.BOINC_DIR="/var/lib/boinc-client"
# This is the location of the BOINC core client, that the init script

uses.
# If you do not want to use the client program provided by the

boinc-client# package, you can specify here an alternative client
program.
# BOINC_CLIENT="/usr/local/bin/boinc

"BOINC_CLIENT="/usr/bin/boinc"
# Here you can specify additional options to pass to the BOINC core

client.
# Type 'boinc --help' or 'man boinc' for a full summary of

allowed options.
# BOINC_OPTS="--allow_remote_gui_rpc

"BOINC_OPTS=""

-- System Information
Debian Release: wheezy/sid  
APT prefers unstable
APT policy: (500, 'unstable'), (500, 'testing'), (500, 'stable'), (1,
'experimental')
Architecture: amd64 (x86_64)

Kernel: Linux 3.1.0-1-amd64 (SMP w/2 CPU cores)
Locale: LANG=en_IN,
LC_CTYPE=en_IN (charmap=UTF-8)
Shell: /bin/sh linked to /bin/dash

Versions of packages boinc-client depends on:
ii  adduser                3.113
ii  ca-certificates        20111025
ii  debconf [ 1.5.41
ii  libc6                  2.13-21
ii  libcurl3               7.23.1-2
ii  libgcc1                1:4.6.2-5
ii  libssl1.0.0            1.0.0e-3
ii  libstdc++6             4.6.2-5
ii  libx11-6               2:1.4.4-4
ii  libxss1                1:1.2.1-2
ii  python                 2.7.2-9
ii  zlib1g                 1:1.2.3.4.dfsg-3

Versions of packages boinc-client recommends:
ii  ia32-libs  20111001

Versions of packages boinc-client suggests:
ii  boinc-app-seti     <none>
ii  boinc-manager      7.0.2+dfsg-1
ii  libcuda1           <none>
ii  libcuda1-ia32      <none>
ii  x11-xserver-utils  7.6+3

-- Configuration Files:/etc/boinc-client/gui_rpc_auth.cfg [Errno 13](debconf-2.0])
Permission denied: u'/etc/boinc-client/gui_rpc_auth.cfg'

-- debconf information:  boinc-client/remove_boinc_dir: false

Migrated-From: http://boinc.berkeley.edu/trac/ticket/1161

**Comment by romw** [comment]
**Commented by shirish on 20 Dec 41935152 16:45 UTC**


Hi all,

I have installed boinc-client but while installing boinc gave couple of warnings.
I have aptitude give a bit more verbose output when installing hence saw those. 
Do not have any idea whether its an important warning or not.


```
D000001: deferred_configure updating conffiles
update-rc.d: warning: boinc-client start runlevel arguments (2 3 4 5) do not match LSB Default-Start values (4 5)
update-rc.d: warning: boinc-client stop runlevel arguments (0 1 6) do not match LSB Default-Stop values (0 1 2
3 6)
```


Can you please look it up and see. Please let me know if any more help
is needed ?

One another thing, I got this privacy message while trying to file the
bug in reportbug about whether I choose to include or not the info.
that boinc-client/remove_boinc_dir: false is set this way or not. I have no
idea what boinc-client/remove_boinc_dir: false does and if that is the
default setup or not.

Looking forward to comments and suggestions.

```
-- Package-specific info:
-- Contents of /etc/default/boinc-client:
# This file is /etc/default/boinc-client, it is a configuration file for
the
# /etc/init.d/boinc-client init script.
# Set this to 1 to enable and to 0 to disable the init script.ENABLED="1"
# Set this to 1 to enable advanced scheduling of the BOINC core client
and all its sub-processes (reduces the impact of BOINC on the
system's performance).
SCHEDULE="1"
# The BOINC core client will be started with the permissions of this
user.
BOINC_USER="boinc"
# This is the data directory of the BOINC core
client.
BOINC_DIR="/var/lib/boinc-client"
# This is the location of the BOINC core client, that the init script
uses.
# If you do not want to use the client program provided by the
boinc-client's package, you can specify here an alternative client
program.
#BOINC_CLIENT="/usr/local/bin/boinc"
BOINC_CLIENT="/usr/bin/boinc"
# Here you can specify additional options to pass to the BOINC core
client.
# Type 'boinc --help' or 'man boinc' for a full summary of
allowed options.
#BOINC_OPTS="--allow_remote_gui_rpc"
BOINC_OPTS=""


-- System Information:
Debian Release: wheezy/sid  APT prefers unstable
APT policy: (500, 'unstable'), (500, 'testing'), (500, 'stable'), (1,
'experimental')
Architecture: amd64 (x86_64)

Kernel: Linux 3.1.0-1-amd64 (SMP w/2 CPU cores)
Locale: LANG=en_IN,
LC_CTYPE=en_IN (charmap=UTF-8)
Shell: /bin/sh linked to /bin/dash

Versions of packages boinc-client depends on:
ii  adduser  3.113
ii  ca-certificates        20111025
ii  debconf [ 1.5.41
ii  libc6                  2.13-21
ii  libcurl3               7.23.1-2
ii  libgcc1                1:4.6.2-5
ii  libssl1.0.0            1.0.0e-3
ii  libstdc++6             4.6.2-5
ii  libx11-6               2:1.4.4-4
ii  libxss1                1:1.2.1-2
ii  python                 2.7.2-9
ii  zlib1g                 1:1.2.3.4.dfsg-3

Versions of packages boinc-client recommends:
ii  ia32-libs  20111001

Versions of packages boinc-client suggests:
ii  boinc-app-seti     <none>
ii  boinc-manager      7.0.2+dfsg-1
ii  libcuda1           <none>
ii  libcuda1-ia32      <none>
ii  x11-xserver-utils  7.6+3

-- Configuration Files:
/etc/boinc-client/gui_rpc_auth.cfg [Errno 13](debconf-2.0])
Permission denied: u'/etc/boinc-client/gui_rpc_auth.cfg'

-- debconf information:
boinc-client/remove_boinc_dir: false
```



This is already filed as http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=651278

**Comment by ChristianBeer** [comment]
According to Debian bugtracker fixed in 7.0.26.
